docs/tools/replace.sh

```diff
@@ -70,21 +82,69 @@ function init() {
 }
 
 function main() {
-    node="$1"
-    src_drive="${2#/dev/}"
-    dest_drive="${3#/dev/}"
-
-    # Get source drive ID
-    src_drive_id=$(get_drive_id "${node}" "${src_drive}")
-    if [ -z "${src_drive_id}" ]; then
-        echo "source drive ${src_drive} on node ${node} not found"
+    src_drive="${1#/dev/}"
+    dest_drive="${2#/dev/}"
+    node="${3}"
+
```

Comment:
You could error out if `src_drive == dest_drive` here. Also error out if `node` argument is missing if `src_drive`/`dest_drive` arguments are drive names.